log-engine-tuulbox/src/commonMain/kotlin/logs/TuulboxLogEngine.kt

```diff
@@ -0,0 +1,30 @@
+package com.juul.kable.logs
+
+import com.juul.tuulbox.logging.HideFromStackTraceTag
+import com.juul.tuulbox.logging.Log
+
+public object TuulboxLogEngine : LogEngine, HideFromStackTraceTag {
```

Comment:
This is my bad, but it occurs to me now that it would be nice to have an `object Kable : Key<Boolean>` so we can set `metadata[Kable] = true` on these log calls.

Not a breaking change to add this later, so if you would rather convert to issue feel free.